selectBoard.php

```diff
@@ -77,6 +78,22 @@
         $stmt = $db->prepare($query); 
        	$result = $stmt->execute($query_params);   	
 
+
+$query = " 
+          INSERT INTO taskList ( 
+              listTitle, 
+              boardID
+          ) VALUES 
+            ('TO-DO',:boardID),('DONE',:boardID)
```

Comment:
The query looks nice! 👍 

Tested by creating a Board: observed the 2 default lists named "TO-DO" and "DONE" are present after the creation of the Board